azkaban-exec-server/src/test/java/azkaban/execapp/fake/FakeApp.java

```diff
@@ -0,0 +1,14 @@
+package azkaban.execapp.fake;
+
+import azkaban.execapp.AzkabanExecutorServer;
+import azkaban.execapp.jmx.JmxJobMBeanManager;
+import azkaban.utils.Props;
+
+public class FakeApp extends AzkabanExecutorServer {
```

Comment:
I prefer Mockito, but it wasn't in the dependencies yet when I started working on this :) I'll make the switch.